include/liblcvm.h

```diff
@@ -21,13 +21,18 @@
 // @param[in] infile: Name of the file to be parsed.
 // @param[out] num_video_frames: Number of video frames in the file.
 // @param[out] frame_rate_fps: Frame rate (fps).
+// @param[out] frame_drop_count: Frame drop count.
 // @param[out] frame_drop_ratio: Frame drop ratio (unitless).
 // @param[out] normalized_frame_drop_average_length: Normalized frame drop
 // length.
-// @param[in] debug: Debug level.
+// @param[in] percentile: Percentile.
+// @param[out] frame_drop_length_percentile: Frame drop length percentile.
+// @param[in] debug: Debug level.    
 int get_frame_drop_info(const char *infile, int *num_video_frames,
-                        float *frame_rate_fps, float *frame_drop_ratio,
-                        float *normalized_frame_drop_average_length, int debug);
+                        float *frame_rate_fps, int *frame_drop_count,
+                        float *frame_drop_ratio, float *normalized_frame_drop_average_length,
+                        float percentile, float *frame_drop_length_percentile,
```

Comment:
Line 34 should be:

                        const std::vector<float> percentile_list, std::vector<float> &frame_drop_length_percentile_list,